pkg/cmd/codespace/common.go

```diff
@@ -57,20 +56,8 @@ func (a *App) StopProgressIndicator() {
 	a.io.StopProgressIndicator()
 }
 
-type liveshareSession interface {
-	Close() error
-	GetSharedServers(context.Context) ([]*liveshare.Port, error)
-	KeepAlive(string)
-	OpenStreamingChannel(context.Context, liveshare.ChannelID) (ssh.Channel, error)
-	StartJupyterServer(context.Context) (int, string, error)
-	StartSharing(context.Context, string, int) (liveshare.ChannelID, error)
-	StartSSHServer(context.Context) (int, string, error)
-	StartSSHServerWithOptions(context.Context, liveshare.StartSSHServerOptions) (int, string, error)
-	RebuildContainer(context.Context, bool) error
-}
-
 // Connects to a codespace using Live Share and returns that session
-func startLiveShareSession(ctx context.Context, codespace *api.Codespace, a *App, debug bool, debugFile string) (session liveshareSession, err error) {
+func startLiveShareSession(ctx context.Context, codespace *api.Codespace, a *App, debug bool, debugFile string) (session *liveshare.Session, err error) {
```

Comment:
Wondering if this should also move the the `liveshare` package?

Reply:
I think that would probably be good to do in the future, but it will introduce a large number of changes so that should probably be done in a later PR.